### PR TITLE
net-news/alligator: new package

### PR DESCRIPTION
**Pull request by CyberTailor**

### CODE REVIEW

File: net-news/alligator/alligator-23.04.49.9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_TEST="forceoptional"
+KFMIN=5.75.0
+QTMIN=5.15.0
```
Comment (a17r) [inline net-news/alligator/alligator-23.04.49.9999.ebuild:8]: Minor: This is not up to same level as other KDE Gear packages for both vars, but would be fixed automatically in future.

**Review by a17r — APPROVED** [review]
Thanks for your work, looks solid. I'll try to get to build, install and check deps soon.

File: net-news/alligator/alligator-23.04.49.9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+LICENSE="|| ( GPL-2 GPL-3 ) CC0-1.0 CC-BY-SA-4.0 GPL-3+"
+SLOT="0"
+KEYWORDS=""
+
+DEPEND="
+	>=dev-libs/kirigami-addons-0.6:5
+	>=dev-qt/qtdeclarative-${QTMIN}:5
+	>=dev-qt/qtgui-${QTMIN}:5
+	>=dev-qt/qtnetwork-${QTMIN}:5
+	>=dev-qt/qtquickcontrols-${QTMIN}:5
```
Comment (a17r) [inline net-news/alligator/alligator-23.04.49.9999.ebuild:23]: I don't see evidence of this being used, not in CMakeLists.txt or qml files.

**Review by a17r — CHANGES_REQUESTED** [review]

File: net-news/alligator/alligator-23.04.49.9999.ebuild (hunk 2 above)
Comment (CyberTailor) [reply net-news/alligator/alligator-23.04.49.9999.ebuild:23]: It's reported by iwdevtools

File: net-news/alligator/alligator-23.04.49.9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_TEST="forceoptional"
+KFMIN=5.75.0
+QTMIN=5.15.0
+inherit ecm gear.kde.org
+
+DESCRIPTION="A convergent RSS/Atom feed reader for Plasma"
```
[the remark this answers is not in the document]
Comment (CyberTailor) [reply net-news/alligator/alligator-23.04.49.9999.ebuild]: Fixed

File: net-news/alligator/alligator-23.04.49.9999.ebuild (hunk 2 above)
Comment (a17r) [reply net-news/alligator/alligator-23.04.49.9999.ebuild:23]: could you paste the output here please?

Comment (CyberTailor) [reply net-news/alligator/alligator-23.04.49.9999.ebuild:23]: lddtree output: https://paste.debian.net/1281264/

Comment (CyberTailor) [reply net-news/alligator/alligator-23.04.49.9999.ebuild:23]: oops, some of them are indeed unused

Comment (CyberTailor) [reply net-news/alligator/alligator-23.04.49.9999.ebuild]: Removed `dev-qt/qtquickcontrols` from deps

Comment (a17r) [reply net-news/alligator/alligator-23.04.49.9999.ebuild]: yes, again only meant to comment on `dev-qt/qtquickcontrols` in particular

**Comment by a17r** [comment]
Thanks for your contribution! You may also have a look at my follow-up commits in kde repository.

**Comment by a17r** [comment]
Keywording: https://bugs.gentoo.org/907242

**Pull request closed without merging**